### PR TITLE
added tests to account_invoice_rounding

### PR DESCRIPTION
**Pull request by charbeljc**
also fixed some `.travis.yml` errors and marked `stock_picking_invoicing` as uninstallable

### CODE REVIEW

**Comment by charbeljc** [comment]
Tests are failing on travis, but they pass on my machine ...

**Comment by pedrobaeza** [comment]
Maybe one of the existing modules on the repository changes the tested behaviour.

**Comment by charbeljc** [comment]
Hum, yes @pedrobaeza, there is something suspect with `account_move_line_sort`:

```
2015-07-01 11:51:22,864 23943 DEBUG openerp_test openerp.tools.yaml_import: In order to test that an invoice is correctly rounded down, I configure an invoice.
2015-07-01 11:51:23,039 23943 DEBUG openerp_test openerp.tools.yaml_import: The returning field 'line_order' from your on_change call 'onchange_partner_id' does not exist either on the object 'account.invoice', either in the view 'account.invoice.supplier.form'
```

Thank you, I will look at this

**Comment by charbeljc** [comment]
Hi @pedrobaeza In fact the culprit was `account_invoice_validation_workflow`. I updated the travis build to run tests for this module in isolation.

File: stock_picking_invoicing/__openerp__.py [hunk 1]
```diff
@@ -29,5 +29,5 @@
     "data": [
         "stock_view.xml",
     ],
-    "installable": True
+    "installable": False
```
Comment (eLBati) [inline stock_picking_invoicing/__openerp__.py]: why?

File: .travis.yml [hunk 2]
```diff
@@ -30,6 +32,7 @@ install:
   - git clone https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - pip install anybox.testing.openerp # needed for testing account_invoice_line_sort
```
Comment (eLBati) [inline .travis.yml:35]: Why is [OCA/account-invoicing 8.0 build](https://travis-ci.org/OCA/account-invoicing/builds/67993591) green even without this package?

File: .travis.yml [hunk 3]
```diff
@@ -20,8 +20,10 @@ env:
   matrix:
   - LINT_CHECK="1"
   - TRANSIFEX="1"
-  - TESTS="1" ODOO_REPO="odoo/odoo"
-  - TESTS="1" ODOO_REPO="OCA/OCB
+  - TESTS="1" ODOO_REPO="odoo/odoo" EXCLUDE="account_invoice_validation_workflow"
+  - TESTS="1" ODOO_REPO="OCA/OCB"   EXCLUDE="account_invoice_validation_workflow"
+  - TESTS="1" ODOO_REPO="odoo/odoo" INCLUDE="account_invoice_validation_workflow" EXCLUDE="stock_picking_invoicing,invoice_fiscal_position_update,account_payment_term_extension,account_invoice_shipping_address,account_invoice_rounding,account_invoice_partner,account_invoice_merge,account_invoice_line_sort,account_invoice_line_description,account_invoice_force_number"
+  - TESTS="1" ODOO_REPO="OCA/OCB"   INCLUDE="account_invoice_validation_workflow" EXCLUDE="stock_picking_invoicing,invoice_fiscal_position_update,account_payment_term_extension,account_invoice_shipping_address,account_invoice_rounding,account_invoice_partner,account_invoice_merge,account_invoice_line_sort,account_invoice_line_description,account_invoice_force_number"
```
Comment (eLBati) [inline .travis.yml]: I think `EXCLUDE` can be avoided when `INCLUDE` is used

File: .travis.yml (hunk 2 above)
Comment (charbeljc) [inline .travis.yml:35]: @eLBati , some tests in the module `account_invoice_line_sort` make use of this package. I guess the build should be red, not green: 

```
ERROR openerp_test openerp.modules.module: Can not `import account_invoice_line_sort`.

Traceback (most recent call last):

File "/home/travis/odoo-8.0/openerp/modules/module.py", line 377, in get_test_modules

mod = importlib.import_module('.tests', modpath)

File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module

__import__(name)

File "/home/travis/build/OCA/account-invoicing/account_invoice_line_sort/tests/__init__.py", line 2, in <module>

from . import test_account_invoice_line_sort

File "/home/travis/build/OCA/account-invoicing/account_invoice_line_sort/tests/test_account_invoice_line_sort.py", line 24, in <module>

from anybox.testing.openerp import SharedSetupTransactionCase

ImportError: No module named anybox.testing.openerp
```

File: stock_picking_invoicing/__openerp__.py (hunk 1 above)
Comment (charbeljc) [inline stock_picking_invoicing/__openerp__.py]: Hum, guess I forgot this after doing some test interaction isolation to restore green status.

File: .travis.yml (hunk 3 above)
Comment (charbeljc) [inline .travis.yml]: thx for the tip, done !

**Comment by guewen** [comment]
:+1: the commits in .travis.yml could be squashed

**Comment by charbeljc** [comment]
Thanks @guewen, your careful reviews are always helpful, I'll squash the commits

File: .travis.yml (hunk 2 above)
Comment (eLBati) [inline .travis.yml:35]: Ok I see

**Comment by eLBati** [comment]
thanks :+1:

**Comment by charbeljc** [comment]
@eLBati many thx for the merge ! one thing less in my backlog, cool ... Have a nice end of week !

**Pull request merged**